src/pinns_pde_solve.jl

```diff
@@ -66,20 +65,20 @@ Arguments:
 abstract type AbstractPINN{isinplace} <: SciMLBase.SciMLProblem end
 
 struct PhysicsInformedNN{isinplace,C,T,P,PH,DER,PE,AL,K} <: AbstractPINN{isinplace}
-  chain::C
-  strategy::T
-  init_params::P
-  phi::PH
-  derivative::DER
-  param_estim::PE
-  additional_loss::AL
-  kwargs::K
-
- @add_kwonly function PhysicsInformedNN{iip}(chain,
+    chain::C
```

Comment:
revert all auto-added whitespace

Reply:
Are you ok with my leaving some of the correct indentation on the file? I thought it was more readable like that. Otherwise, I will revert it :)